consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/request/UpdateType.java

```diff
@@ -1,4 +1,8 @@
 package uk.gov.moj.sdt.cmc.consumers.request;
 
-public interface CMCRequest {
+public enum UpdateType {
+
+    MP,
+    WD,
+    DI
```

Comment:
Might be worth adding a comment describing what these values represent (MP = Paid, WD = Settled and DI = Discontinued).

Reply:
done